.gitignore

```diff
@@ -27,6 +27,7 @@ certs/v2_key
 config/secrets.yml*
 coverage/
 Gemfile.lock*
+!Gemfile.lock.release
```

Comment:
Can you put this into a master PR as well?  Then we don't need it special on the branches